frontend/src/components/dashboard/ActivityRow.vue

```diff
@@ -108,6 +116,15 @@ export default {
   mixins: [dateHelperUTCFormatted],
   props: {
     scheduleEntry: { type: Object, default: () => ({ _meta: { loading: true } }) },
+    loadingEndpoints: {
+      type: [Boolean, Object],
```

Comment:
In which case is `loadingEndpoints` assigned a Boolean value?

Reply:
If you don't want to have the loading endpoints behavior at all and don't want to set all endpoints to false.

Reply:
Couldn't you just set the value to `{}` in that case?